src/View/Helper/UrlHelper.php

```diff
@@ -195,11 +218,14 @@ protected function _encodeUrl($url)
      * a timestamp will be added.
      *
      * @param string $path The file path to timestamp, the path must be inside WWW_ROOT
+     * @param bool|string $stamp If set will overrule the value of `Asset.timestamp` in Configure.
      * @return string Path with a timestamp added, or not.
      */
-    public function assetTimestamp($path)
+    public function assetTimestamp($path, $stamp = null)
```

Comment:
We can't change the method arguments in a point/bugfix release.

Reply:
You'll need to target your changes to `3.next` branch and they can be included in `3.6.0`.

Reply:
I guess I did something wrong. Is it better to close this pull request and make a new one?

Reply:
You can rebase your changes onto 3.next and the diff/commit list will be correct.

Reply:
Thanks for pointing me to the right direction.